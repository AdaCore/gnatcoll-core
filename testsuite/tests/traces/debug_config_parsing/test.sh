echo "== DEBUG.LOCATION=yes"
./test .gnatdebug_location

echo ""
echo "== DEBUG.ENCLOSING_ENTITY=yes"
./test .gnatdebug_enclosing_entity

echo ""
echo "== DEBUG.COLORS=yes"
./test .gnatdebug_colors
