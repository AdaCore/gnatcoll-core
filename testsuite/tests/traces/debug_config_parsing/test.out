== DEBUG.LOCATION=yes
[PKG] Entering Test.Foo:test.adb:13 (loc:test.adb:13)
   [PKG] A = 3 (loc:test.adb:15)
   [PKG] Entering Test.Foo:test.adb:13 (loc:test.adb:13)
      [PKG] A = 2 (loc:test.adb:15)
      [PKG] Entering Test.Foo:test.adb:13 (loc:test.adb:13)
         [PKG] A = 1 (loc:test.adb:15)
      [PKG] Leaving Test.Foo:test.adb:13 (loc:test.adb:13)
   [PKG] Leaving Test.Foo:test.adb:13 (loc:test.adb:13)
[PKG] Leaving Test.Foo:test.adb:13 (loc:test.adb:13)

== DEBUG.LOCATION=no
[PKG] Entering Test.Foo:test.adb:13
   [PKG] A = 3
   [PKG] Entering Test.Foo:test.adb:13
      [PKG] A = 2
      [PKG] Entering Test.Foo:test.adb:13
         [PKG] A = 1
      [PKG] Leaving Test.Foo:test.adb:13
   [PKG] Leaving Test.Foo:test.adb:13
[PKG] Leaving Test.Foo:test.adb:13

== DEBUG.ENCLOSING_ENTITY=yes
[PKG] Entering Test.Foo:test.adb:13 (entity:Test.Foo)
   [PKG] A = 3 (entity:Test.Foo)
   [PKG] Entering Test.Foo:test.adb:13 (entity:Test.Foo)
      [PKG] A = 2 (entity:Test.Foo)
      [PKG] Entering Test.Foo:test.adb:13 (entity:Test.Foo)
         [PKG] A = 1 (entity:Test.Foo)
      [PKG] Leaving Test.Foo:test.adb:13 (entity:Test.Foo)
   [PKG] Leaving Test.Foo:test.adb:13 (entity:Test.Foo)
[PKG] Leaving Test.Foo:test.adb:13 (entity:Test.Foo)

== DEBUG.ENCLOSING_ENTITY=no
[PKG] Entering Test.Foo:test.adb:13
   [PKG] A = 3
   [PKG] Entering Test.Foo:test.adb:13
      [PKG] A = 2
      [PKG] Entering Test.Foo:test.adb:13
         [PKG] A = 1
      [PKG] Leaving Test.Foo:test.adb:13
   [PKG] Leaving Test.Foo:test.adb:13
[PKG] Leaving Test.Foo:test.adb:13
